fern/pages/fern-docs/config/navigation.mdx

```diff
@@ -99,16 +92,30 @@ navigation:
               - page: Likelihood
                 path: ./docs/pages/likelihood.mdx
 ```
-</CodeBlock>
 
 ![Result of above docs.yml example](https://fern-image-hosting.s3.amazonaws.com/fern/nested-sections.png)
 
+## Sidebar icons
+
+For icons to appear next to sections and pages, add the `icon` key. The value should be a valid [FontAwesome icon](https://fontawesome.com/icons) name.
+
+```yaml Example navigation config with icons

```

Comment:
TIL you can put the title in the code block itself.

Reply:
<img width="786" alt="Screenshot 2024-05-08 at 2 11 30 PM" src="https://github.com/fern-api/fern/assets/693032/40866170-ad3a-4e98-9056-69caea7cab11">